test/ContentValidationListenerTest.php

```diff
@@ -1914,7 +1986,7 @@ function (MvcEvent $e) use (&$hasRun) {
     public function indexedFields()
     {
         return [
-            'flat-array'   => [['foo', 'bar']],
+            'flat-array'   => [[['foo'], ['bar']]],
```

Comment:
Just a note, for anybody who's research leads here...

This change was necessary because of zendframework/zend-inputfilter#116. That patch introduces features to validate that the data provided to a `CollectionInputFilter` (a) is an array or traversable, and (b) each item is itself also an array or traversable. This prevents some odd side effects in `isValid()` and `getUnknown()` when an invalid collection and/or set within the collection is provided.

However, it also meant that this datum in the data provider was malformed, as each item in the set needed to be an array in order to correctly test the functionality.